marvin.py

```diff
@@ -532,6 +558,21 @@ def delrule(self, update):
 
             return
 
+    def pin_if_necessary(self, to_pin, submission):
+        """ (Telegram command)
+        Pin reddit post if necessary
+        :param to_pin: the message to pin
+        :param submission: the reddit post
+        """
+        for autopin_rule in self.auto_pinned_posts:
+            if str(submission.title.lower()).find(autopin_rule["text"]) != -1:
+                for authors_pin in autopin_rule["user"]:
+                    if (authors_pin == submission.author.name.lower()):
+                        self.updater.bot.pin_chat_message(to_pin.chat_id, to_pin.message_id, disable_notification=True)
```

Comment:
Come mai niente notifica?

Reply:
non saprei, come utente telegram mi viene impensabile stare in un gruppo che ogni 2 giorni fa una notifica pin, muterei il bot, che non è una cosa a cui vorremmo aspirare. Sono scelte, però se fen0x vuole le notifiche mettero = false lì

cc @fen0x